content/pt-br/docs/reference/using-api/client-libraries.md

```diff
@@ -0,0 +1,81 @@
+
+
+<!-- body -->
+Para desenvolver aplicações usando a [API REST do Kubernetes](/docs/reference/using-api/),
+você não precisa implementar as chamadas da API e os tipos de solicitação/resposta. 
+Você pode usar uma biblioteca cliente para a linguagem de programação que está usando.
+
+As bibliotecas clientes geralmente lidam com tarefas comuns para você, como autenticação.
+A maioria das bibliotecas de clientes podem fazer a descoberta e usar a Conta de Serviço do Kubernetes para
+se autenticar, se o cliente API estiver sendo executado dentro do cluster do Kubernetes, ou pode
+entender o formato do [arquivo kubeconfig](/docs/tasks/access-application-cluster/configure-access-multiple-clusters/)
+para ler as credenciais e o endereço do Servidor da API.
+
+## Bibliotecas cliente Kubernetes oficialmente suportadas
+
+As seguintes bibliotecas clientes são oficialmente mantidas pela
+[Kubernetes SIG API Machinery](https://github.com/kubernetes/community/tree/master/sig-api-machinery).
+
+
+| Language   | Client Library | Sample Programs |
+|------------|----------------|-----------------|
+| C          | [github.com/kubernetes-client/c](https://github.com/kubernetes-client/c/) | [browse](https://github.com/kubernetes-client/c/tree/master/examples)
+| dotnet     | [github.com/kubernetes-client/csharp](https://github.com/kubernetes-client/csharp) | [browse](https://github.com/kubernetes-client/csharp/tree/master/examples/simple)
+| Go         | [github.com/kubernetes/client-go/](https://github.com/kubernetes/client-go/) | [browse](https://github.com/kubernetes/client-go/tree/master/examples)
+| Haskell    | [github.com/kubernetes-client/haskell](https://github.com/kubernetes-client/haskell) | [browse](https://github.com/kubernetes-client/haskell/tree/master/kubernetes-client/example)
+| Java       | [github.com/kubernetes-client/java](https://github.com/kubernetes-client/java/) | [browse](https://github.com/kubernetes-client/java#installation)
+| JavaScript | [github.com/kubernetes-client/javascript](https://github.com/kubernetes-client/javascript) | [browse](https://github.com/kubernetes-client/javascript/tree/master/examples)
+| Perl       | [github.com/kubernetes-client/perl/](https://github.com/kubernetes-client/perl/) | [browse](https://github.com/kubernetes-client/perl/tree/master/examples)
+| Python     | [github.com/kubernetes-client/python/](https://github.com/kubernetes-client/python/) | [browse](https://github.com/kubernetes-client/python/tree/master/examples)
+| Ruby       | [github.com/kubernetes-client/ruby/](https://github.com/kubernetes-client/ruby/) | [browse](https://github.com/kubernetes-client/ruby/tree/master/examples)
+
+## Bibliotecas clientes mantidas pela comunidade
+
+{{% thirdparty-content %}}
+
+As seguintes bibliotecas de clientes da API do Kubernetes são fornecidas e mantidas por 
+seus autores, não pela equipe do Kubernetes.
+
+| Language             | Client Library                           |
```

Comment:
Missing translation on this header too.

Reply:
Hello, my name is Daniela, I saw that you have a review to do so I decided to correct the review you made.

Reply:
Hi there, @daniso123!

I'm not sure what you mean. We've been awaiting for this feedback to be addressed by @abner423 so we can move the contribution forward.

Reply:
I believe he won't make the corrections so I want to fix it to finally close this issue.

Reply:
I made the corrections in this issue: [pt-br] Translate client-libraries.md #39194